src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/AutoEnchanter.java

```diff
@@ -92,6 +93,7 @@ protected void tick(Block b) {
 			slots:
 			for (int slot: getInputSlots()) {
 				ItemStack target = BlockStorage.getInventory(b).getItemInSlot(slot == getInputSlots()[0] ? getInputSlots()[1]: getInputSlots()[0]);
+				if(SlimefunItem.getByItem(target) != null && !SlimefunItem.getByItem(target).isEnchantable()) return;
```

Comment:
Heavy call right there.

Should be split into a variable for performance reasons.